### PR TITLE
Adding Topology notification category and ServerAlreadyEnabled

### PR DESCRIPTION
**Pull request by gfx54b**
https://trello.com/c/fQuZB6YA/114-update-to-notification-topology-category-an-repeated-enable-cordon

### CODE REVIEW

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -1227,6 +1228,47 @@ Suggestions for improvement::
...
+.Enabling an already enabled server
+====
+Command::
++
+[source, cypher]
+----
+ENABLE SERVER "123e4567-e89b-12d3-a456-426614174000"
+----
+
+Full title of the returned code::
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1262]: I am a bit confused. Why do we need a title and a full title? Currently, notifications have a title and a description of the returned code. I think these lines should compose the description.

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -1227,6 +1228,47 @@ Suggestions for improvement::
...
+Server '123e4567-e89b-12d3-a456-426614174000' was already enabled.
+Make sure nothing is misspelled.
+
+Description of the returned code::
+The server was already enabled, calling this command has no effect.
```
[the remark this answers is not in the document]
Comment (Lojjs) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1267]: In this case, I think it is probably not the spelling that is wrong but a copy-paste error or similar as that id is quite different between servers I imagine?

It is different if you tried to do something on a server that did not exist, then you might have spelled it wrong.

I would therefore rather write something like  the `Verify that this is the intended server.` below than `Verify that the spelling is correct.`

File: modules/ROOT/pages/notifications/all-notifications.adoc (hunk 1 above)
Comment (Lojjs) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1262]: Yes in 5.x there is one title and one description, while in 6.x there will be a message so we should think this split through a bit more

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 3]
```diff
@@ -1227,6 +1228,47 @@ Suggestions for improvement::
...
+.Enabling an already enabled server
+====
+Command::
++
+[source, cypher]
+----
+ENABLE SERVER "123e4567-e89b-12d3-a456-426614174000"
+----
+
+Full title of the returned code::
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1262]: It should look something similar to this one:

Screenshot 2023-09-28 at 10 58 59

**Review by renetapopova — COMMENTED** [review]
I added some suggestions. Could you see if that makes sense to you as well?

File: modules/ROOT/pages/notifications/all-notifications.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (gfx54b) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1262]: Okay, sorry, I was not sure, what to copy and what not to. This makes sense, thank you!

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 4]
```diff
@@ -1227,6 +1228,43 @@ Suggestions for improvement::
 Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 ====
 
+[#_topology_notifications]
+== `TOPOLOGY` category
+
+Topology notifications provide additional information related to managing databases and servers.
+databases and/or servers has some additional information to tell the user.
```
Comment (Lojjs) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1235]: Now this line start mid+sentence, should it be removed as well?

Comment (gfx54b) [reply modules/ROOT/pages/notifications/all-notifications.adoc]: Already done! :smile:

**Review by renetapopova — APPROVED** [review]
Looks good to me! Thanks, @gfx54b.

**Comment by gfx54b** [comment]
So let's wait with the merge until the implementation PR is in a mergeable state. Now we target 5.13 - but if that PR gets delayed, it will got into 5.14 only.
Also we don't have a verdict if the category `TOPOLOGY` will be added.

**Comment by jackwaudby** [comment]
Leaving this here as a reminder - as we have changed the notification to `ServerAlready*` the associated docs PR needs an update

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 5]
```diff
@@ -1227,6 +1228,41 @@ Suggestions for improvement::
...
+Topology notifications provide additional information related to managing databases and servers.
+
+[#_neo_clientnotification_cluster_unnecessaryenableserver]
+=== UnnecessaryEnableServer
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Cluster.UnnecessaryEnableServer
```
Comment (gfx54b) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1243]:
```suggestion
m|Neo.ClientNotification.Cluster.ServerAlreadyEnabled
```

File: modules/ROOT/pages/changelogs.adoc [hunk 6]
```diff
@@ -1,6 +1,15 @@
 :description: This page lists all changes to status codes per Neo4j version.
 = Changes to status codes per Neo4j version
 
+== Neo4j 5.13
+
+**New:**
+
+[source, status codes, role="noheader"]
+-----
+Neo.ClientNotification.Cluster.UnnecessaryEnableServer
```
Comment (gfx54b) [inline modules/ROOT/pages/changelogs.adoc:10]:
```suggestion
Neo.ClientNotification.Cluster.ServerAlreadyEnabled
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 7]
```diff
@@ -1227,6 +1228,41 @@ Suggestions for improvement::
 Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 ====
 
+[#_topology_notifications]
+== `TOPOLOGY` category
+
+Topology notifications provide additional information related to managing databases and servers.
+
+[#_neo_clientnotification_cluster_unnecessaryenableserver]
+=== UnnecessaryEnableServer
```
Comment (gfx54b) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1237]:
```suggestion
=== ServerAlreadyEnabled
```

**Comment by gfx54b** [comment]
@renetapopova @Lojjs Apologies about the many emails. @jackwaudby suggested a change for the constant. Then instead of doing the sensible thing of changing everything at once offline, I've tried to update the PR in github. This ended up me forgetting stuff. Hopefully it is all right now. 
Please shout if you don't like the new `ServerAlreadyEnabled` constant.

**Pull request merged**